media/js/base/mozilla-monitor-button.js

```diff
@@ -12,27 +12,28 @@ if (typeof window.Mozilla === 'undefined') {
 
     var MonitorButton = {};
 
-    MonitorButton.init = function(buttonId){
+    MonitorButton.init = function(){
+        var buttons;
 
-        // check for custom buttonId, used in case of multiple buttons.
-        if(!buttonId){
-            buttonId = 'fxa-monitor-submit';
-        }
+        // Collect all monitor buttons
+        buttons = document.getElementsByClassName('js-monitor-button');
 
-        var monitorButton = document.getElementById(buttonId);
+        // Exit if no valid button in DOM
+        if (buttons[0] === undefined) {
+            return;
+        }
 
-        // fetch request
+        // Exit if no fetch support
         var supportsFetch = 'fetch' in window;
-
-        if (!supportsFetch || !monitorButton) {
+        if (!supportsFetch) {
             return;
         }
 
-        var buttonURL = monitorButton.getAttribute('href');
+        var buttonURL = buttons[0].getAttribute('href');
```

Comment:
Hmm, so this always gets the first match of a button in the page, which is then used to overwrite the `href` for every button in the page. Would this wipe out button specific params such as `utm_content=primary|secondary`?

If so, it feels like we should be preserving the original parameters, rather than overwriting. Thoughts?

Reply:
Also: it would be great to add a unit test for this new functionality, to make sure it's doing the right things?

Reply:
If these values aren't important to pass to the fetch request, then can we remove the code that includes them to begin with? https://github.com/mozilla/bedrock/blob/master/media/js/base/mozilla-monitor-button.js#L45-L56